internal/npm_install/test/golden/node_modules/@gregmagolan/test-a/BUILD.bazel.golden

```diff
@@ -1,29 +1,29 @@
 
 package(default_visibility = ["//visibility:public"])
 load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
-filegroup(
+load("@build_bazel_rules_nodejs//internal/npm_install:node_module_library.bzl", "node_module_library")
+node_module_library(
     name = "test-a__pkg",
     srcs = [
         ":test-a__files",
+    ],
+    deps = [
```

Comment:
nit: minify delta by not writing empty deps

Reply:
done